snuba_sdk/legacy.py

```diff
@@ -134,10 +135,17 @@ def to_scalar(value: Any) -> Any:
         limit, name = limitby
         query = query.set_limitby(LimitBy(Column(name), int(limit)))
 
-    extras = ("limit", "offset", "granularity", "totals")
+    extras = (
+        "limit",
+        "offset",
+        "granularity",
```

Comment:
In the legacy system, the granularity is set by the timeseries extension.
https://github.com/getsentry/snuba/blob/master/snuba/query/timeseries_extension.py#L87

Could you confirm this is performed by the snql parser now that the extension is not executed anymore ?

Reply:
Yes the SnQL parser sets it on the query. https://github.com/getsentry/snuba/blob/master/snuba/query/snql/parser.py#L206